bitnami/influxdb/templates/influxdb/configmap-backup.yaml

```diff
@@ -2,26 +2,60 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "influxdb.fullname" . }}-backup
+  name: {{ include "common.names.fullname" . }}-backup
   labels:
-    {{- include "influxdb.labels" . | nindent 4 }}
+    {{- include "common.labels.standard" . | nindent 4 }}
     app.kubernetes.io/component: influxdb
 data:
   backup.sh: |-
     #!/bin/bash
 
     set -e
 
+    . /opt/bitnami/scripts/libinfluxdb.sh
+
     DATE="$(date +%Y%m%d_%H%M%S)"
+    BRANCH="$(influxdb_branch)"
+
+    host="{{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc"
+
+    get_orgs() {
+      INFLUX_TOKEN="${INFLUXDB_ADMIN_USER_TOKEN}" influx --host "http://${host}:{{ .Values.influxdb.service.port }}" org list 2> /dev/null | grep -v 'ID' | awk -F '\t' 'BEGIN{ORS=" "} {print $2}'
+    }
+
+    get_databases() {
+        local org_name="${1:-}"
+        if [[ "${BRANCH}" = "1" ]]; then
+            influx -username "{{ .Values.auth.admin.username }}" -password "${INFLUXDB_ADMIN_USER_PASSWORD}" -host "${host}" -port {{ .Values.influxdb.service.port }} -execute 'SHOW DATABASES' | sed -e '1,3d'
+        else
+            INFLUX_TOKEN="${INFLUXDB_ADMIN_USER_TOKEN}" influx --host "http://${host}:{{ .Values.influxdb.service.port }}" bucket list --org "${org_name}" 2> /dev/null | grep -v 'ID' | awk -F '\t' 'BEGIN{ORS=" "} {print $2}'
```

Comment:
Is there anyway to tell `influx` cli to format the output in JSON or YAML so you can use `jq` and you avoid using grep, sed, awk, etc.

Reply:
Yes, it is, we would need to add this tool to the image then, do you think it is worth it? `awk` solves the problem and it simple.

Reply:
I think `jq` is less error-prone rather than using `awk`/`grep`, but I don't have a strong opinion about this.